cmd/collector/app/handler_test.go

```diff
@@ -48,9 +49,17 @@ func (p *possiblyErroringJaegerBatchesHandler) SubmitBatches(ctx tchanThrift.Con
 	return nil, p.err
 }
 
+type possiblyErroringZipkinBatchesHandler struct {
```

Comment:
I think the correct verb form is "erring" from "to err", but why not just call this mockXxxHandler?